src/management-system-v2/components/version-toolbar.tsx

```diff
@@ -1,112 +1,57 @@
 'use client';
 
-const {
+import {
   setDefinitionsId,
   setDefinitionsName,
   manipulateElementsByTagName,
   generateDefinitionsId,
-} = require('@proceed/bpmn-helper');
+  getUserTaskFileNameMapping,
+} from '@proceed/bpmn-helper';
 
-import React, { useEffect, useState } from 'react';
+import React, { useState } from 'react';
 
 import type ElementRegistry from 'diagram-js/lib/core/ElementRegistry';
 
-import { Tooltip, Button, Space, Modal, Form, Input } from 'antd';
+import { Tooltip, Button, Space, Modal } from 'antd';
 
 import { FormOutlined, PlusOutlined } from '@ant-design/icons';
 
 import useModelerStateStore from '@/lib/use-modeler-state-store';
-import { useParams } from 'next/navigation';
-import { FormInstance } from 'antd/es/form';
-import { useGetAsset, post } from '@/lib/fetch-data';
-
-const ModalSubmitButton = ({ form, onSubmit }: { form: FormInstance; onSubmit: Function }) => {
-  const [submittable, setSubmittable] = useState(false);
-
-  // Watch all values
-  const values = Form.useWatch([], form);
-
-  React.useEffect(() => {
-    form.validateFields({ validateOnly: true }).then(
-      () => {
-        setSubmittable(true);
-      },
-      () => {
-        setSubmittable(false);
-      },
-    );
-  }, [form, values]);
+import { useParams, useRouter } from 'next/navigation';
+import { get, del, put, usePostAsset } from '@/lib/fetch-data';
+import { convertToEditableBpmn } from '@/lib/helpers/processVersioning';
+import { asyncForEach, asyncMap } from '@/lib/helpers/javascriptHelpers';
+import ProcessCreationButton from './process-creation-button';
 
-  return (
-    <Button
-      type="primary"
-      htmlType="submit"
-      disabled={!submittable}
-      onClick={() => {
-        onSubmit(values);
-        form.resetFields();
-      }}
-    >
-      Create Process
-    </Button>
-  );
-};
-
-type ProcessModalProps = {
+type ConfirmationModalProps = {
   show: boolean;
-  close: (values?: { name: string; description: string }) => void;
+  close: () => void;
+  confirm: () => void;
 };
-const ProcessModal: React.FC<ProcessModalProps> = ({ show, close }) => {
-  const [form] = Form.useForm();
-
+const ConfirmationModal: React.FC<ConfirmationModalProps> = ({ show, close, confirm }) => {
   return (
     <Modal
-      title="Create new Process"
```

Comment:
Modal close button on the top right gets in the way of the text, maybe `closeIcon={null}` is a good solution
![image](https://github.com/PROCEED-Labs/proceed/assets/5623598/188cae1d-c0c8-45e5-93da-54141e5c4ec5)